pyproject.toml

```
@@ -24,7 +24,7 @@ classifiers = [
 requires-python = ">=3.9"
 
 dependencies = [
-    "aiida-core~=2.0",
+    "aiida-core>=2.7",
```

Better add upper bound.
```suggestion
    "aiida-core>=2.7, <3",
```

---

Won't `aiida-core~=2.7` also do the trick?

https://peps.python.org/pep-0440/#compatible-release